src/hooks/engine.rs

```diff
@@ -780,6 +793,17 @@ pub static V_FadeAlpha: Pointer<unsafe extern "C" fn() -> c_int> = Pointer::empt
     ]),
     my_V_FadeAlpha as _,
 );
+pub static V_RenderView: Pointer<unsafe extern "C" fn()> = Pointer::empty_patterns(
+    b"V_RenderView\0",
+    // To find, search for "(0x4000)". This is the input for `glClear`. Cycle through results
```

Comment:
Like, text in disassembly? Sounds weird as a search target, maybe there's a more reasonable way?

Reply:
To find, search for "R_RenderView: NULL worldmodel". This is an output error for `R_RenderView()`. Then, find a function calling `R_RenderView()` exactly 3 times. That function will be `V_RenderView()`.

Alternative it, I could mention the `glClear` input instead because there are three functions calling `R_RenderView()` but only `V_RenderView()` calling some functions with argument `0x4000`